CycloneDX/Services/GithubService.cs

```diff
@@ -68,8 +68,8 @@ public class GithubService : IGithubService
         private HttpClient _httpClient;
         private List<Regex> _githubRepositoryRegexes = new List<Regex>
         {
-            new Regex(@"^https?\:\/\/github\.com\/(?<repositoryId>[^\/]+\/[^\/]+)\/((blob)|(raw))\/(?<refSpec>[^\/]+)\/[Ll][Ii][Cc][Ee][Nn][Ss][Ee]((\.|-)((md)|([Tt][Xx][Tt])|([Mm][Ii][Tt])|([Bb][Ss][Dd])))?$"),
```

Comment:
I know this is just an update to an existing regex.
But what about using RegexOptions.IgnoreCase to simplify?